charge.js

```diff
@@ -0,0 +1,58 @@
+var parseArgs = require('minimist')
+var fs = require('fs')
+var path = require('path')
+var pkg = require('./package.json')
+var exporter = require('./index')
+
+var argOptions = {
```

Comment:
While looking into how to pass boolean arguments I saw that minimist supported these options and it looked a lot cleaner than all the conditions scattered about.